torch/onnx/utils.py

```diff
@@ -655,6 +659,16 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
                 attrs = {k: n[k] for k in n.attributeNames()}
                 return symbolic_fn(g, *inputs, **attrs)
 
+        elif ns == "quantized":
+            if operator_export_type == OperatorExportTypes.ONNX_ATEN_FALLBACK:
```

Comment:
Can we generalize this logic? If it's ONNX_ATEN_FALLBACK, you can set the domain to 'caffe2'. Otherwise, keep it ''